SEC/Debt/Bonds.rdf

```diff
@@ -1456,6 +1496,15 @@
 		<skos:definition>indicates whether a legal opinion exists for a given municipal bond</skos:definition>
 	</owl:DatatypeProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-sec-dbt-bnd;isLinkedToFallback">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+		<rdfs:label>is linked to fallback</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;FinancialInstrument"/>
+		<rdfs:range rdf:resource="&fibo-sec-dbt-bnd;GovernmentBond"/>
+		<skos:definition>relates an index-linked instrument to a government bond that may be selected by a calculation agent</skos:definition>
```

Comment:
neither the definition nor the note say anything about "fallback" which is crucial i,.e. that the rate of the fallback bond is used as a substitute for the inflation index if, for example, it is no longer published.

Reply:
Added your note to the explanatory note.